src/webapps/azext_webapps/__init__.py

```diff
@@ -0,0 +1,32 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azure.cli.core import AzCommandsLoader
+import azext_webapps._help  # pylint: disable=unused-import
+
+
+class WebappsCommandLoader(AzCommandsLoader):
```

Comment:
Would recommend `WebappsExtCommandLoader` instead.
The azure-cli-appservice module has the name `AppserviceCommandsLoader` and just want to make sure that the class names don't conflict in the future. By adding `Ext`, this should be clear enough.